packages/okta-vue/package.json

```diff
@@ -8,15 +8,33 @@
     "src"
   ],
   "scripts": {
+    "prebuild": "npm run build:package-info",
     "prestart": "npm run build",
     "pretest": "npm run build && npm run build:harness",
     "prepublish": "npm run build",
-    "test": "npm run --prefix test/e2e/harness/ test",
+    "jest": "jest src/",
+    "test": "npm run jest && npm run --prefix test/e2e/harness/ test",
```

Comment:
I'm leaning towards thinking we should prepend `npm run lint` to this command. That way we can start enforcing the `eslint` style.